test/functional/editions_controller_test.rb

```diff
@@ -264,6 +264,46 @@ class EditionsControllerTest < ActionController::TestCase
     end
   end
 
+  context "#duplicate" do
+    setup do
+      @answer = FactoryBot.create(:answer_edition, panopticon_id: FactoryBot.create(:artefact).id)
+    end
+
+    should "redirect to edit tab and show success message when user saves successfully" do
+      EditionDuplicator.any_instance.expects(:duplicate).returns(true)
+      EditionDuplicator.any_instance.expects(:new_edition).returns(@answer)
+      post :duplicate, params: { id: @edition.id, to: "help_page" }
+
+      assert_response :found
```

Comment:
Maybe we should assert on the redirect to match the test name?
```suggestion
      assert_redirected_to edition_path(@answer)
```